server/lib/request-handler.js

```diff
@@ -1,9 +1,70 @@
-import { wordAssocAPIk } from './api_config.js';
+//import { wordAssocAPIk } from './api_config.js';
+var apiKey = require('./api_config.js');
 var path = require('path');
 var unirest = require('unirest');
-var db = require('/../../db');
+var User = require('./../../db/models/user');
+var util = require('./utility.js');
+//var db = require('/../../db');
 
+/////////////////////////////////////////////////////////////////
+//AUTHENTICATION
 
+exports.userLogin = function(req, res) {
+  var username = req.body.username;
+  var password = req.body.password;
+
+  User.findOne( {username: username} )
+    .exec(function(err, user) {
```

Comment:
Need to handle (err) from the db query (eg. error accessing DB), (and not just when username has length === 0).